firstvoices/backend/tests/factories/resources/import_job/test_invalid_dictionary_entries.csv

```diff
@@ -0,0 +1,6 @@
+﻿title,type,visibility,acknowledgement,include_in_games,include_on_kids_site
+Word 1,xyz,Public,123,TRUE,FALSE
+Phrase 1,phrase,Team,123,FALSE,TRUE
+,notAType,Public,123,FALSE,TRUE
```

Comment:
Row 3 doesn't have any obvious issue to me, while row 4 has two issues (empty title and notAType). Is that correct? just checking.

Reply:
Yeah, nice catch. That row is fine, I just left it there as a control.